SConstruct

```diff
@@ -52,7 +52,7 @@ DESIRED_FLAGS = ['-Wno-gnu-designator',
                  '-Wno-gnu-zero-variadic-macro-arguments',
                  '-Wno-tautological-constant-out-of-range-compare']
 
-CART_VERSION = "1.1.0"
+CART_VERSION = "1.2.0"
```

Comment:
This is minor version change.

Reply:
We are already at 1.2.0 right now in master, so this needs to be rolled further to 1.3.0
Also requires changes to cart.spec and debian/changelog files

Reply:
done